resources/newssite/news-next/src/context/data-context.js

```diff
@@ -13,6 +28,20 @@ const strings = {
         buttons: buttonsEn,
         links: linksEn,
     },
+    jp: {
+        content: contentJp,
+        settings: settingsJp,
+        footer: footerJp,
+        buttons: buttonsJp,
+        links: linksJp,
+    },
+    ar: {
```

Comment:
can we do what I was suggesting earlier? Automatically using `rtl` if `ar` is specified?
As a reminder we're using this list in my project: https://github.com/firefox-devtools/profiler/blob/main/src/app-logic/l10n.js#L57
but I'd be happy enough if you shortlist that to just `ar` currently.

(both for next and nuxt)

Reply:
Happy to add